### PR TITLE
feat: add eval task authoring phase to service-reference agent

### PR DESCRIPTION
**Pull request by ahmadabdalla**
## Summary

- Adds **Phase 7: Author the Eval Task** to `.github/agents/service-reference.md`, making eval task creation a mandatory step before commit
- Renames the existing Phase 7 (Commit and PR) to **Phase 8** and adds eval task path and `waza check` result to the required PR description items

## Problem

The agent had no instruction to create an eval task. It would open a PR, CI coverage check would fail, and the agent would either stall or reactively fix it — a fragile loop.

## Design

Phase 7 is self-contained with no new sub-agent. Eval task authoring is template-driven (fixed tags, standard grader set, deterministic file path) and does not require the investigation/consensus machinery used for service references. The template covers:

- File path derivation from category and service name
- Scenario selection rules (East US for USD, at least one quantified parameter, no disambiguation needed)
- Full YAML template with all required fields and graders
- Authoring rules for edge cases: consumption-billed services (omit `"730"`), `hasMeters: false` services (replace `uses-pricing-script` grader)
- `waza check` validation gate before proceeding

## Test plan

- [ ] Read Phase 7 against an existing task YAML (e.g., `tasks/compute/app-service/linux-p1-v3.yaml`) and verify the template would produce equivalent output
- [ ] Confirm `{service-name}` in the `service:` tag section matches the pattern used in `check-eval-coverage.sh`
- [ ] Confirm Phase 8 PR description list now includes the eval task checkpoint

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Release Notes

* **Documentation**
  * Updated service contribution workflow to require eval task creation before committing
  * Added detailed specifications for task file structure and required metadata, including tags and graders
  * Introduced schema validation step to ensure task compliance
  * Updated PR checklist to include eval task submission and validation confirmation

### CODE REVIEW

File: docs/ops/evals.md [hunk 1]
```diff
@@ -149,19 +149,77 @@ Runs real AI evaluations. Auth priority: `COPILOT_GITHUB_TOKEN` > `GH_TOKEN` > `
...
+| `trigger`          | Skill activation/deactivation                        | Yes                                                                                  |
+| `skill_invocation` | Correct skill invoked                                | Yes                                                                                  |
+| `prompt`           | LLM-as-judge for qualitative checks                  | No; set task-level `timeout_seconds: 30` to prevent mock hangs                       |
+| `code` (planned)   | Python assertions for numeric accuracy               | Yes                                                                                  |
 
 ### Standard graders for happy-path tasks
 
-Every happy-path task must include the `uses-pricing-script` grader. The Bash tool reports as `bash` in session tool names — not as the script filename. `behavior.required_tools` cannot match script names and silently passes without asserting anything. Use `tool_constraint` instead:
+Every happy-path task uses the following five graders. The first four are fixed; the fifth (`{parameter}-acknowledged`) is parameterised per service.
```
[the remark this answers is not in the document]
Comment (ahmadabdalla) [reply docs/ops/evals.md]: Fixed in 801e8de — rewrote the intro to reflect what the task corpus actually shows: 3 required graders (correct-service-name, currency-format, uses-pricing-script) plus at least one of {parameter}-acknowledged or complete-estimate-quality per service. The 'five graders' framing was wrong.
